entities/communication/voiceCall.json

```diff
@@ -0,0 +1,70 @@
+{
+  "type": "object",
+  "properties": {
+    "uid": {
+      "type": "string",
+      "description": "Unique identifier for the call"
+    },
+    "duration": {
+      "type": "int",
+      "description": "Duration of the call in seconds"
+    },
+    "created_at": {
+      "type": "timestamp",
+      "description": "Time the call was created"
+    },
+    "updated_at": {
+      "type": "timestamp",
+      "description": "Time the call was last updated"
+    },
+    "status": {
+      "type": "enum",
+      "values": [
+        "incoming call",
+        "answered by staff",
+        "missed",
+        "answered by vonage",
+      ],
+      "description": "Status of the call"
+    },
+    "direction": {
+      "type": "enum",
+      "values": [
+        "inbound/outbound"
+      ],
+      "description": "Whether the call is inbound or outbound"
+    },
+    "staff_uid": {
+      "type": "string",
+      "description": "Unique identifier for the staff member who answered the call"
+    },
+    "client_uid": {
+      "type": "string",
+      "description": "Unique identifier for the client who made the call"
+    },
+    "from_number_id": {
+      "type": "string",
+      "description": "Unique identifier for the number the call was made from"
+    },
+    "to_number_id": {
+      "type": "string",
+      "description": "Unique identifier for the number the call was made to"
+    },
+    "recording": {
+      "type": "string",
+      "description": "URL to the recording of the call"
+    },
+    "summary": {
+      "type": "string",
+      "description": "Text summary of the call"
+    },
+    "external_app_link": {
+      "type": "string",
+      "description": "URL to an external app page with more details about the call"
```

Comment:
Link to an application page with additional details about the call.

The phrase external is not accurate. It will eventually may lead to an internal iframe page which is not really an external page